### PR TITLE
Fix some typos in first few chapters

### PR DESCRIPTION
**Pull request by artimath**
Hey mate,

Found some typos while reading through. Noted them down. Saw this repo and figured... why not PR?

Some I couldn't fix, or they were ambiguous, so I will include the screenshots so you can check yourself.

I will continue to note any of these down as I progress through the book.

### CODE REVIEW

**Comment by isovector** [comment]
Mate, you're incredible. Cheers!

**Comment by artimath** [comment]
screenshots of the various typos, or potential typos.

B6AF5F8A-8D8E-45FD-99D3-0999BCE6D37F_1_105_c
8F89E366-622D-40C0-88BD-E7B815B43D32_1_105_c
2B602EBC-4BDF-4CCA-AE19-E0541DEADE81_1_105_c
9A646FA7-01F1-43A8-847C-EC82517BD9D2_1_105_c
3F8F6130-8A18-44EC-AD75-D29550D334D5_1_105_c
ECCA171B-3C25-4207-AD96-B83DD8D6078E_1_105_c
85A58B31-E09E-469F-9F28-4A747483736A_1_105_c
2402B873-EC7F-4B5F-9F57-32E71BDCB8D4_1_105_c
AEE5E787-035B-4D11-838F-D184C4C58C51_1_105_c
8D93A18A-00C1-4149-980A-6F26D1C70BFC_1_105_c
01B696A4-4AF7-4959-BA58-B030BD8791BC_1_105_c
41AF015F-D49C-4EF0-ACD2-C00BCB7F1B06_1_105_c
F243EF4D-0E6C-49E2-A791-00C4996C2C35_1_105_c
A23771A4-1F36-469D-85DB-13F5C9895B8A_1_105_c
67639851-974C-4F38-9C3D-6B60E54DF8C1_1_105_c
E0404BA1-4F51-43AF-BC4A-2BAD2B56C03E_1_105_c
255F5C65-19BE-4310-94F4-E21AF5A29670_1_105_c
085C7AA6-7F12-410A-85D3-B2B49A389383_1_105_c
FE55EDA6-65E7-4F88-BB6D-FC3D2843A409_1_105_c
7EC40CA9-40F2-48CB-BC38-47FC5FE8DBEE_1_105_c
38A1663C-635E-410A-B8D5-69897817C19E_1_105_c
DABDEFC4-47A4-44FD-A99D-26A7FE71CCE2_1_101_o

File: src/book/Chapter1-Agda.lagda.md [hunk 1]
```diff
@@ -1143,7 +1143,7 @@ Exercise
 : The `def:_∨_` function corresponds to the boolean operation OR, which is
   `ctor:true` if either of its arguments was. There is an analogous function
   `def:_∧_` (input via [and](AgdaMode)), which returns `ctor:true` if *both*
-  only when both of its arguments are. Define this function.
+ of its arguments are. Define this function.
```
Comment (isovector) [inline src/book/Chapter1-Agda.lagda.md:1146]:
```suggestion
  of its arguments are. Define this function.
```

**Comment by artimath** [comment]
my pleasure. If you want to send me a up to date pdf, I can continue reading through that one, and will share any typos I find . I am going through the leanpub version downloaded as of last week. Not sure how recent that is and I haven't compiled your hacked pandoc yet to be able to run tbe make.

Ryan Hunter

**Comment by isovector** [comment]
Hi @artimath, I would *adore* any more proofreading you might be able to throw my way. An updated version is available on leanpub :) Thank you so much!!

**Pull request merged**